src/PhpGenerator/Method.php

```diff
@@ -107,6 +118,7 @@ public function __toString()
 			. ($this->name ? ' ' . $this->name : '')
 			. '(' . implode(', ', $parameters) . ')'
 			. ($this->uses ? ' use (' . implode(', ', $uses) . ')' : '')
+			. ($returnType ? ' : ' .$returnType : '')
```

Comment:
I am not sure what is better to use, `function x() : abc` or `function x(): abc`.